.github/workflows/test.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 18.x, 20.x]
```

Comment:
Please don't drop a tested version in a feature PR.

Reply:
Removed related commit.